patches/chrome-browser-themes-theme_service_win.h.patch

```diff
@@ -0,0 +1,21 @@
+diff --git a/chrome/browser/themes/theme_service_win.h b/chrome/browser/themes/theme_service_win.h
+index eba1cb2596d71b91e06183209503f3fc83b0e593..edd9fb7d718f1e3b7c203fdd2936430a7eff38c3 100644
+--- a/chrome/browser/themes/theme_service_win.h
++++ b/chrome/browser/themes/theme_service_win.h
+@@ -20,7 +20,16 @@ class ThemeServiceWin : public ThemeService {
+  private:
+   // ThemeService:
+   bool ShouldUseNativeFrame() const override;
++#if defined(BRAVE_CHROMIUM_BUILD)
```

Comment:
we don't have to patch this, we can use an override to make ThemeServiceWin a subclass of BraveThemeService